### PR TITLE
dependencies-add

### PR DESCRIPTION
**Pull request by jmfayard**
- Add Material Components Compose theme adapter dependency constant #250
- Add Koin, Http4k, Kodein #238

### CODE REVIEW

**Comment by jmfayard** [comment]
What kind of rules should I add? I don't know how this works.
Also it would be cool to have a checklist in `CONTRIBUTING.md` of how to contribute new dependencies like this @LouisCAD

**Comment by jmfayard** [comment]
There is also a rule missing for *spek*
I had to do it manually in the playground-repo

```
// versions.properties
version.org.spekframework.spek2..spek-dsl-jvm=version.spek

version.org.spekframework.spek2..spek-runner-junit5=version.spek

version.org.spekframework.spek2..spek-runtime-jvm=version.spek

version.spek=2.0.13
```

File: plugins/dependencies/src/main/kotlin/dependencies/Http4k.kt [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+
+    /**
+     * [JSON RPC](https://www.http4k.org/guide/modules/jsonrpc/)
+     */
+    const val jsonrpc = "$prefix-jsonrpc"
+
+    /**
+     * [Metrics](https://www.http4k.org/guide/modules/metrics/)
+     */
+    const val metrics_micrometer = "$prefix-metrics-micrometer"
```
Comment (LouisCAD) [inline plugins/dependencies/src/main/kotlin/dependencies/Http4k.kt:49]: Why do we have snake_case here while we have camelCase elsewhere?

**Comment by LouisCAD** [comment]
@jmfayard Could you split this PR per library family? That will make review easier and will enable quicker integration for the libraries I'm familiar with.

**Pull request closed without merging**